src/node_contextify.cc

```diff
@@ -236,7 +236,13 @@ class ContextifyContext {
     Local<String> script_source(args[0]->ToString(args.GetIsolate()));
     if (script_source.IsEmpty())
       return;  // Exception pending.
-    Context::Scope context_scope(Debug::GetDebugContext());
+
+    Environment* env = Environment::GetCurrent(args.GetIsolate());
```

Comment:
It will account for 0% of cases, because debug context is empty only when we are hard on memory (i.e. allocation failure).